### ISSUE TITLE
The image may be upside down after being compressed

### ISSUE THREAD

**Issue by durianboy** [Lymphatus/caesium-image-compressor#290]
**Describe the bug**
Photos taken with mobile phones sometimes become upside down after being compressed.

**Software version**
v2.7.1

**Operating System information**
Windows 11 23H2

**To Reproduce**
1. Drag the picture to the software
2. Set lossless and resize to fit percentage (width: 25%, height: 25%)
3. Click Compress

**Expected behavior**
Images should not be rotated after compression.

**Screenshots**
Before compression
![P_20240716_144957](https://github.com/user-attachments/assets/16b5a5e1-c50d-45e2-aebb-468c91a8f6ca)
![P_20240518_143109](https://github.com/user-attachments/assets/fa9a59d0-587d-4b6b-9b51-b6eb4ad67507)

After compression
![P_20240716_144957壓縮](https://github.com/user-attachments/assets/b0a6c4d3-a27c-4d0a-8dc5-5a5bb3fd10f7)
![P_20240518_143109壓縮](https://github.com/user-attachments/assets/0c710f88-2c33-44c1-94f7-46e45fa601b8)

**Application Log**
[caesium-2024-07-18.log](https://github.com/user-attachments/files/16277127/caesium-2024-07-18.log)

**Additional context**
Rotating compressed photos using Microsoft File Explorer increases file size🥲

**Comment by Lymphatus** [comment]
Selecting "Keep Metadata" in the compression tab should keep the rotation intact.

**Comment by durianboy** [comment]
That's right! Selecting "Keep Metadata" will prevent rotation, but could this be a separate option? Some people may not want to hold metadata but want to keep the image oriented correctly. And I'm curious why this only seems to happen when compressing phone photos.
